packages/manager-react-components/src/components/tiles/BillingInformationsTile/BillingDetails.class.ts

```diff
@@ -0,0 +1,235 @@
+import { isAfter, isBefore } from 'date-fns';
+import { ServiceDetails } from '../../../hooks/services/services.type';
+
+export default class BillingDetails {
```

Comment:
Why use class syntax ?
It just looks like it makes it harder to maintain and read

Reply:
The main reason it's because this is the code in AngularJS. And I tried to keep the maximum of this logic. And I tried to do the better with the time allocated and keep the maximum same method to keep comparison possible